package.json

```diff
@@ -24,7 +26,9 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.13"
+    "ethers-multicall": "^0.2.3",
+    "forta-agent": "^0.1.13",
+    "forta-agent-tools": "^3.1.2"
```

Comment:
In this bot the `forta-agent-tools` is only used for testing, so it should be moved to `devDependencies`